### DISCUSSION TITLE
Grid-Config in "Search objects" appears twice

### DISCUSSION THREAD

**Discussion by polymed** [pimcore/pimcore#8481] [Q&A / Support] [answered]
Hi guys,

What can be the reason if a view which definitely exists only once in the DB appears twice?

Best regards

![Bildschirmfoto 2021-03-16 um 16 46 18](https://user-images.githubusercontent.com/51081904/111338390-31d1df00-8677-11eb-89cf-5976aca22ae5.png)

**Comment by dvesh3** [comment, marked as answer]
seems related https://github.com/pimcore/pimcore/issues/7016
